### PR TITLE
ISSUE-860 UI Add timeout option to topology test run

### PR DESCRIPTION
**Pull request by joylyn**
UI side changes for #860

### CODE REVIEW

File: webservice/src/main/resources/app/scripts/containers/Streams/TopologyEditor/TopologyEditorContainer.jsx [hunk 1]
```diff
@@ -1480,8 +1495,15 @@ class TopologyEditorContainer extends Component {
         </Modal>
 
         {/*ConfirmBox to Run TestCase*/}
-        <Modal ref="confirmRunTestModal" data-title="Confirm Box" dialogClassName="confirm-box" data-resolve={this.confirmRunTest.bind(this, true)} data-reject={this.confirmRunTest.bind(this, false)}>
-          {<p> Are you sure you want to run the test case?</p>}
+        <Modal ref="confirmRunTestModal" data-title="Are you sure you want to run the test case?" data-resolve={this.confirmRunTest.bind(this, true)} data-reject={this.confirmRunTest.bind(this, false)}>
```
Comment (shahsank3t) [inline webservice/src/main/resources/app/scripts/containers/Streams/TopologyEditor/TopologyEditorContainer.jsx]: nit: Change the title message to "Are you sure you want to run the test case with the following configuration?"

**Comment by shahsank3t** [comment]
+1 over all looks good. Just a nit to change the title message.

**Comment by joylyn** [comment]
Review changes added.

**Comment by HeartSaVioR** [comment]
+1

**Pull request merged**